pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
```

Comment:
Here is an example with poetry if you want to avoid setuptools https://github.com/Porchetta-Industries/CrackMapExec/blob/master/pyproject.toml

Reply:
@noraj thanks for the suggestion. I've just removed setuptools and opt'd for poetry. Let me know if this works

Reply:
Working for me.